README

```diff
@@ -1,7 +1,9 @@
 Python Twitter Tools
 ====================
 
-[![Build Status](https://travis-ci.org/sixohsix/twitter.svg)](https://travis-ci.org/sixohsix/twitter)
+![Test]()
```

Comment:
what is this line for?

Reply:
That's a mistake, will remove it.

I'd pasted the new badge boilerplate there, and copied and pasted out the bits we needed, then forgot to tidy up. Thanks for checking!

Reply:
Removed!